docs/reference/release-notes/8.0.0-alpha2.asciidoc

```diff
@@ -3,4 +3,77 @@
 
 coming::[8.0.0-alpha2]
```

Comment:
> coming::[8.0.0-alpha2]

If you can remove this "coming" flag in this PR too, that'll save us an extra PR.

Reply:
done https://github.com/jakelandis/elasticsearch/commit/1d1b0f612f5f3aacd04481501f4daceec161dcee